components/match2/wikis/overwatch/match_group_input_custom.lua

```diff
@@ -355,7 +355,17 @@ function matchFunctions.getOpponents(match)
 
 			-- Retrieve icon for team
 			if opponent.type == Opponent.team then
-				opponent.icon = opponentFunctions.getIcon(opponent.template)
+				if opponent.template:lower() == 'bye' then
+					opponent = {type = Opponent.literal, name = 'BYE'}
+				else
+					opponent.icon = opponentFunctions.getIcon(opponent.template)
+				end
+			elseif opponent.type ~= Opponent.literal then
+				-- process other opponent types (especially in regards to their player inputs)
+				-- ..........
```

Comment:
actually do the processing if there is something to be done, else kick the comment
check if players are actually already available at this point